packages/shared/src/domain/address/zip.ts

```diff
@@ -1,17 +1,43 @@
 import { MetriportError } from "../../error/metriport-error";
 
-function isValidZipCode(zipCode: string): boolean {
+export const zipLength = 5;
+
+export const knownInvalidZipCodes = [
+  "00000",
+  "99999",
+  "10000",
+  "20000",
+  "30000",
+  "40000",
+  "50000",
+  "60000",
+  "70000",
+  "80000",
+  "90000",
+  "12345",
+  "54321",
+];
+
+// TODO This should be more comprehensive and include a check for the length of the zip code
+// and the contents (4324- should fail)
+export function isValidZipCode(zipCode: string): boolean {
   if (!zipCode) return false;
-  if (zipCode.length === 0) return false;
+  // TODO Ideally we'd call isValidZipCodeLength here, but this would be a breaking change at this point
+  if (zipCode.length < 1) return false;
   if (!zipCode.match(/^[0-9-]+$/)) return false;
+  if (knownInvalidZipCodes.some(invalidZipCode => zipCode === invalidZipCode)) return false;
+  return true;
+}
+
+export function isValidZipCodeLength(zipCode: string): boolean {
+  if (zipCode.length !== zipLength) return false;
   return true;
 }
```

Comment:
doesn't this fail on zip plus 4s?

Reply:
no, it doesn't. I don't want to change the behavior w/ this PR, just check for invalid zips.

Reply:
ahh i see function names had me turned around. yeah i see the regex [0-9-]+ now